examples/http/build.sbt

```diff
@@ -12,7 +12,8 @@ lazy val root = (project in file("."))
     libraryDependencies ++= Seq(
       "io.circe" %%% "circe-core",
       "io.circe" %%% "circe-parser"
-    ).map(_ % "0.9.0")
+    ).map(_ % "0.9.0"),
+    libraryDependencies += "org.julienrf" %%% "scalm" % "1.0.0-RC1+20180118-2216" // todo: remove and figure out how to to better testing
   )
 
-lazy val scalm = RootProject(uri("git://github.com/julienrf/scalm.git"))
+//lazy val scalm = RootProject(uri("git://github.com/julienrf/scalm.git"))
```

Comment:
by curiosity, what was the problem with the commented solution?

Reply:
You can try that:

~~~ scala
lazy val scalm = ProjectRef(file("..", "scalm"))
~~~

Reply:
works :) (only it must be `../..`)